tests/FSharp.Data.Tests/JsonValue.fs

```diff
@@ -416,4 +426,4 @@ let ``Encoding of markup is not overzealous`` () =
   let w = new IO.StringWriter()
   JsonValue.JsonStringEncodeTo w input
   let expected = input // Should not escape </>
-  (w.GetStringBuilder().ToString()) |> should equal expected
\ No newline at end of file
+  (w.GetStringBuilder().ToString()) |> should equal expected
```

Comment:
Hm, could you please remove the newline afterwards? It's unnecessary here 😕

Reply:
@realvictorprm, this is done.